[RTL] Add option to apply built-in effects to the individual connected glyphs.

bruvzg opened the pull request:
Adds `connected=0/1` option to `shake`, `tornado` and `wave` effects to control if it's applied to individual glyphs.

https://user-images.githubusercontent.com/7645683/190925564-0dba73ce-f37e-4892-a43e-6f957b6bc0ce.mov

Fixes https://github.com/godotengine/godot/issues/66058

bruvzg committed:
[RTL] Add option to apply built-in effects to the individual connected glyphs.

akien-mga commented:
Is there a precedent for these `t`/`f` abbreviations? It doesn't seem so clear to me reading such syntax, I'd expect `1`/`0` if we need a short form, or full `true`/`false`.

`con` also doesn't seem very explicit as a name, since it's likely going to be a rare option I think it could be a full `connected` (or `join` if we want an option that stays short).

Would also be good if @3booodpro could confirm that's the behavior they expect. I'm afraid that they might want a full stretching deformation of the glyphs while keeping them connected, which would be orders of magnitude more complex than this.

bruvzg commented:
> Is there a precedent for these t/f abbreviations? It doesn't seem so clear to me reading such syntax, I'd expect 1/0 if we need a short form, or full true/false.

No, changed it to `0/1` int, and renamed to `connected`.

> I'm afraid that they might want a full stretching deformation of the glyphs while keeping them connected, which would be orders of magnitude more complex than this.

That would probably require something like `TextMesh` deformation to keep it always connected, definitely not for the `RichTextLabel` effects.

3booodpro commented:
> Would also be good if @3booodpro could confirm that's the behavior they expect. I'm afraid that they might want a full stretching deformation of the glyphs while keeping them connected, which would be orders of magnitude more complex than this.

Nah this is good enough, keeping them connected and deforming the stretching will take so effort for now..
but definitely it will be a really cool enhancement for the future but for now its good enough!

akien-mga commented:
Thanks!
